infra/azure/templates/group_tests.yml

```diff
@@ -1,46 +1,70 @@
 ---
 parameters:
-  - name: scenario
+  - name: distro
     type: string
     default: fedora-latest
   - name: build_number
     type: string
   - name: ansible_version
     type: string
     default: ""
+  - name: target_python
```

Comment:
Please try to get rid of target_python. The platform or system python should be selected automatically by Ansible.

Reply:
Hopefully fixed. (Pipelines are running for ~8H now).